pkg/addons/default/aws_node_test.go

```diff
@@ -52,59 +51,6 @@ var _ = Describe("default addons - aws-node", func() {
 			Expect(awsNode.Spec.Template.Spec.Containers[0].Image).To(
 				Equal("602401143452.dkr.ecr.eu-west-2.amazonaws.com/amazon-k8s-cni:v1.3.2"),
 			)
-
-		})
-
-		It("can update 1.10 sample to latest", func() {
```

Comment:
It looks like there is no similar test based on 1.11 or another version, we should migrate this code.

Reply:
See https://github.com/weaveworks/eksctl/pull/1029/files#diff-6ddb8dc16176800c04be724aaff37219L24

I'd start by changing it to:
```
sampleAddons := testutils.LoadSamples("testdata/sample-1.12.json")
```

And then make the rest of the test relevant.

Reply:
> make the rest of the test relevant

This is the hard bit, given I know little about this logic and the background of this change, but I've had a go.